app/src/main/java/com/dsu/poc/MainActivity.kt

```diff
@@ -1,6 +1,13 @@
 package com.dsu.poc
 
+import android.os.Bundle
 import com.dsu.poc.base.BaseActivity
+import com.dsu.poc.project.ui.login.LoginFragment
 
 class MainActivity : BaseActivity() {
```

Comment:
add class level java documentation and function level java documentation with parameter description

Reply:
done